src/ImageRunModal.jsx

```diff
@@ -837,8 +837,19 @@ export class ImageRunModal extends React.Component {
                             </Flex>
                         </FormGroup>
 
-                        {(this.state.owner === systemOwner || (this.state.image && this.state.image.isSystem)) &&
-                            <FormGroup fieldId='run-image-cpu-priority' label={_("CPU shares")}>
+                        {this.state.owner === systemOwner &&
+                            <FormGroup
+                              fieldId='run-image-cpu-priority'
+                              label={_("CPU shares")}
+                              labelIcon={
+                                  <Popover aria-label={_("CPU Shares help")}
+                                      enableFlip
+                                      bodyContent={_("CPU shares determine how much CPU time a container can use relative to other running containers. Set it to a lower number for lower priority or a higher number for higher.")}>
```

Comment:
IMHO the second sentence is rather redundant, and makes the string harder to translate. Can we trim this a bit?

    The more shares a container has, the more CPU time it gets relative to other running containers.

Reply:
How's this?

```suggestion
                                      bodyContent={_("CPU shares determine priority of running containers. Default priority is 1024. Higher numbers prioritize containers. Lower numbers decrease priority.")}>
```

Reply:
We already shown the default as 1024 as greyed out, should we mention it in the information popover a well.

Reply:
It does help to explain what's going on.

Reply:
Ok,`Higher numbers prioritize containers` sounds a bit vague to me, as one could read it as prioritizes all containers.

Reply:
How about singular? "A higher number prioritizes this container"?

Reply:
Sounds good, I made the suggested changes.